components/collection/drop/Generative.vue

```diff
@@ -89,33 +49,20 @@
 </template>
 
 <script setup lang="ts">
-import UnlockableCollectionInfo from '@/components/collection/unlockable/UnlockableCollectionInfo.vue'
-import UnlockableTag from '@/components/collection/unlockable/UnlockableTag.vue'
-import CarouselTypeLatestMints from '@/components/carousel/CarouselTypeLatestMints.vue'
 import { createUnlockableMetadata } from '../unlockable/utils'
-import GenerativePreview from '@/components/collection/drop/GenerativePreview.vue'
 import { DropItem } from '@/params/types'
-import { DoResult, doWaifu } from '@/services/waifu'
+import { doWaifu } from '@/services/waifu'
 import { useDropMinimumFunds, useDropStatus } from '@/components/drops/useDrops'
-import { makeScreenshot } from '@/services/capture'
-import { pinFileToIPFS } from '@/services/nftStorage'
-import { sanitizeIpfsUrl } from '@/utils/ipfs'
-import newsletterApi from '@/utils/newsletter'
-import { prefixToToken } from '@/components/common/shoppingCart/utils'
 import DropConfirmModal from './modal/DropConfirmModal.vue'
 import ListingCartModal from '@/components/common/listingCart/ListingCartModal.vue'
 import { nftToListingCartItem } from '@/components/common/shoppingCart/utils'
 import { fetchNft } from '@/components/items/ItemsGrid/useNftActions'
-import { DEFAULT_NEWSLETTER_SUBSCRIPTION } from '@/stores/preferences'
+import useGenerativeDropMint from './composables/useGenerativeDropMint'
+import useGenerativeDropNewsletter from './composables/useGenerativeDropNewsletter'
+import useGenerativeDropDetails from './composables/useGenerativeDropDetails'
```

Comment:
should we keep it like this?

- link https://nuxt.com/docs/guide/directory-structure/composables#composables-directory

Reply:
done